src/main/java/Storage.java

```diff
@@ -0,0 +1,111 @@
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+
+/**
+ * The {@code Storage} class stores and retrieves tasks.
+ */
+public class Storage {
+
+    private File file;
+
+    /**
+     * Initialises a Storage object by creating a new file or retrieving
+     * an existing file if possible.
+     */
+    public Storage() {
+        Path path = Paths.get("data/");
```

Comment:
Maybe can consider save the path in a constant? It makes it easier to access in write to file method (instead of calling file.getPath())

Reply:
LGTM, thanks for advice.